components/Comment/lib/quill.css

```diff
@@ -0,0 +1,308 @@
+.ql-container {
```

Comment:
This file is copy of old quill file. Going to make changes and do not want  to affect old comment module